components/AddDeposit/styles.js

```diff
@@ -3,4 +3,5 @@ import styled from "styled-components";
 const AddDepositHeading = styled.h1`
   margin-top: 100px;
 `;
+
 export { AddDepositHeading };
```

Comment:
why are you exporting in brackets? :) 
```suggestion
export  AddDepositHeading ;
```